app/views/user/CertificatesView.coffee

```diff
@@ -10,13 +10,17 @@ ThangTypeConstants = require 'lib/ThangTypeConstants'
 ThangType = require 'models/ThangType'
 utils = require 'core/utils'
 fetchJson = require 'core/api/fetch-json'
+locale = require 'locale/locale'
 
 module.exports = class CertificatesView extends RootView
   id: 'certificates-view'
   template: require 'templates/user/certificates-view'
+  nowLang: me.get('preferredLanguage', true)
```

Comment:
Should we instantiate this in `intialize`?
I don't think we should be setting this on a prototype of the `CertificatesView`.